apps/nrm/views/fr-details.html

```diff
@@ -0,0 +1,17 @@
+{{<partials-page}}
+  {{$page-content}}
+    <div class="govuk-form-group">
+        <label class="govuk-label">
+        Email
+        </label>
+        <span class="govuk-hint">
+        This is the address you gave at the start of the form.
+        </span>
+        <p class="govuk-body">FR EMAIL PLACEHOLDER</p>
+    </div>
+    {{#renderField}}fr-details-name{{/renderField}}
+    {{#renderField}}fr-details-role{{/renderField}}
+    {{#renderField}}fr-details-phone{{/renderField}}
```

Comment:
will this not just generate if it's in `index.js` i.e. https://github.com/UKHomeOffice/modern-slavery/blob/master/apps/nrm/index.js

Reply:
Actioned in the PR for MS-724 What are your contact details?